src/commands/BootstrapCommand.js

```diff
@@ -27,7 +40,7 @@ export default class BootstrapCommand extends Command {
     this.progressBar.init(this.packages.length);
     this.logger.info("Linking all dependencies");
 
-    async.parallelLimit(this.packages.map(pkg => done => {
+    async.parallelLimit(this.packages.filter(pkg => !pkg.ignored).map(pkg => done => {
```

Comment:
I would rather structure this differently where we don't pass down ignore packages into the package graph and instead we do the check here if the package has been ignored.

This should remove a lot of the complexity of this feature.